public/app/plugins/panel/table/renderer.ts

```diff
@@ -264,6 +264,15 @@ export class TableRenderer {
       cellClasses.push('table-panel-cell-pre');
     }
 
+    if (column.style && column.style.align) {
+      cellStyles.push(`text-align:${column.style.align}`);
```

Comment:
If the user has chosen to format as `Number` and not chosen any `alignment` then maybe the `default` should be `right`? wdyt @torkelo?

Reply:
Not changing this for the time being.